From ec65a15061b0615c142edb5b704a8976ef5550d4 Mon Sep 17 00:00:00 2001
From: Hannes Janetzek <hannes.janetzek@gmail.com>
Date: Tue, 9 May 2017 15:47:34 +0200
Subject: save


diff --git a/Makefile.am b/Makefile.am
index afe2c02..aa97e77 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -66,27 +66,27 @@ bin_PROGRAMS = \
 prime_serverd_SOURCES = \
         src/prime_serverd.cpp
 prime_serverd_CPPFLAGS = $(DEPS_CFLAGS)
-prime_serverd_LDADD = $(DEPS_LIBS) libprime_server.la
+prime_serverd_LDADD = $(DEPS_LIBS) libprime_server.la -lpthread
 prime_httpd_SOURCES = \
         src/prime_httpd.cpp
 prime_httpd_CPPFLAGS = $(DEPS_CFLAGS)
-prime_httpd_LDADD = $(DEPS_LIBS) libprime_server.la
+prime_httpd_LDADD = $(DEPS_LIBS) libprime_server.la -lpthread
 prime_proxyd_SOURCES = \
         src/prime_proxyd.cpp
 prime_proxyd_CPPFLAGS = $(DEPS_CFLAGS)
-prime_proxyd_LDADD = $(DEPS_LIBS) libprime_server.la
+prime_proxyd_LDADD = $(DEPS_LIBS) libprime_server.la -lpthread
 prime_workerd_SOURCES = \
         src/prime_workerd.cpp
 prime_workerd_CPPFLAGS = $(DEPS_CFLAGS)
-prime_workerd_LDADD = $(DEPS_LIBS) libprime_server.la
+prime_workerd_LDADD = $(DEPS_LIBS) libprime_server.la -lpthread
 prime_echod_SOURCES = \
         src/prime_echod.cpp
 prime_echod_CPPFLAGS = $(DEPS_CFLAGS)
-prime_echod_LDADD = $(DEPS_LIBS) libprime_server.la
+prime_echod_LDADD = $(DEPS_LIBS) libprime_server.la -lpthread
 prime_filed_SOURCES = \
 	src/prime_filed.cpp
 prime_filed_CPPFLAGS = $(DEPS_CFLAGS)
-prime_filed_LDADD = $(DEPS_LIBS) libprime_server.la
+prime_filed_LDADD = $(DEPS_LIBS) libprime_server.la -lpthread
 
 # tests
 check_PROGRAMS = test/zmq test/netstring test/http test/shaping test/interrupt
